packages/components-library/src/components/ui/Pagination.vue

```diff
@@ -0,0 +1,287 @@
+<template>
+<nav class="c-pagination">
+
+  <div class="button-container">
+
+    <button v-if="val.count > val.size" :class="styles" class="t-page-prev btn btn-outline-primary mr-4"
+      :disabled="val.page <= 1"
+      @click="navigate(val.page - 1)">
+        &laquo;
+    </button>
+
+    <div class="btn-group">
+      <template v-if="ellipsisFirst">
+        <button :class="styles" class="btn btn-outline-primary" @click="navigate(ellipsisFirst)">
+            {{ellipsisFirst}}
+        </button>
+        <button :class="styles" class="btn ellipsis" disabled>...</button>
+      </template>
+
+      <button
+        class="btn btn-outline-primary"
+        :class="{'active': pageNumber === val.page, ...styles}" v-for="pageNumber in pages" :key="pageNumber"
+        @click="navigate(pageNumber)">
+        {{pageNumber}}
+      </button>
+
+      <template v-if="ellipsisLast">
+        <button :class="styles" class="btn ellipsis" disabled>...</button>
+        <button :class="styles" class="btn btn-outline-primary" @click="navigate(ellipsisLast)">
+          {{ellipsisLast}}
+        </button>
+      </template>
+
+      <button v-if="val.count > val.size"
+        :class="styles" class="t-page-next btn btn-outline-primary ml-4"
+        :disabled="val.page >= pageCount"
+        @click="navigate(val.page + 1)">
+        &raquo;
+      </button>
+    </div>
+
+    <form v-if="val.count > val.size" class="pagination-controls form-inline ml-2">
+      <div class="btn-group">
+        <select  class="form-control-sm ml-2 mr-2" v-model="val.size">
+          <option value="10">10</option>
+          <option value="20">20 </option>
+          <option value="50">50</option>
+        </select>
+        <label for="exampleInputEmail1">per page</label>
```

Comment:
maybe expose via prop or use i18n to change / translate label

Reply:
Added as prop